wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/en.md

```diff
@@ -1,20 +1,20 @@
 # Code of conduct for modding and mapping
 
-The **Code of Conduct for Modding and Mapping** is a set of rules and guidelines that apply to the entirety of the osu! [modding](/wiki/Modding) and [mapping](/wiki/Beatmapping) ecosystem and an extension to the [osu! community rules](/wiki/Rules) and [general ranking criteria](/wiki/Ranking_Criteria). In order to ensure that mapping and modding discussions in threads take place in a constructive, positive and productive environment, a code of conduct is crucial to get everyone on the same page. When participating in the modding and mapping ecosystem, it is mandatory to follow this Code of Conduct. Misconduct that violates these rules might lead to [penalties issued to your account](/wiki/Silence).
+The **Code of conduct for modding and mapping** is a set of rules and guidelines that apply to the entirety of the osu! [modding](/wiki/Modding) and [mapping](/wiki/Beatmapping) ecosystem and an extension to the [osu! community rules](/wiki/Rules) and [general ranking criteria](/wiki/Ranking_Criteria). In order to ensure that mapping and modding discussions in threads take place in a constructive, positive and productive environment, a code of conduct is crucial to get everyone on the same page. When participating in the modding and mapping ecosystem, it is mandatory to follow this code of conduct. Misconduct that violates these rules might lead to [penalties issued to your account](/wiki/Silence).
 
-## Behaviour and Conduct
+## Behaviour and conduct
 
-- The goal of modding is to help a mapper improve their work so that it can be pushed towards [Ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
+- The goal of modding is to help a mapper improve their work so that it can be pushed towards [ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
```

Comment:
```suggestion
- The goal of modding is to help a mapper improve their work so that it can be pushed towards the [ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
```